### ISSUE TITLE
Non-producing notice shows up multiple times

### ISSUE THREAD

**Issue by Jolly-Pirate** [steemit/steem#2892]
The `Not producing block because I don't have the private key for` notice shows up 3-5 times consecutively in 0.20.0.

```
3597283ms p2p_plugin.cpp:212            handle_block         ] Got 45 transactions on block 25895739 by aggroed -- latency: 283 ms
566ms p2p_plugin.cpp:212            handle_block         ] Got 41 transactions on block 25895740 by lukestokes.mhth -- latency: 566 ms
2602ms witness_plugin.cpp:527        block_production_loo ] Not producing block because I don't have the private key for STM7dkef3P39JFEv1RhnVU85vWdETUS5eWA2LGiaALLfLji5kdAt8
2800ms witness_plugin.cpp:527        block_production_loo ] Not producing block because I don't have the private key for STM7dkef3P39JFEv1RhnVU85vWdETUS5eWA2LGiaALLfLji5kdAt8
3000ms witness_plugin.cpp:527        block_production_loo ] Not producing block because I don't have the private key for STM7dkef3P39JFEv1RhnVU85vWdETUS5eWA2LGiaALLfLji5kdAt8
3200ms witness_plugin.cpp:527        block_production_loo ] Not producing block because I don't have the private key for STM7dkef3P39JFEv1RhnVU85vWdETUS5eWA2LGiaALLfLji5kdAt8
3400ms witness_plugin.cpp:527        block_production_loo ] Not producing block because I don't have the private key for STM7dkef3P39JFEv1RhnVU85vWdETUS5eWA2LGiaALLfLji5kdAt8
3413ms p2p_plugin.cpp:212            handle_block         ] Got 36 transactions on block 25895741 by drakos -- latency: 413 ms
6613ms p2p_plugin.cpp:212            handle_block         ] Got 56 transactions on block 25895742 by followbtcnews -- latency: 613 ms
```

**Comment by mvandeberg** [comment]
I am assuming the bug report is that this message is showing up multiple times, not because it is showing up at all.

We made some changes to the block production loop. There is a production window and a wake frequency for the production loop. We increased the window slightly and increased the wake frequency because in extreme cases the node could miss the production window entirely. Each message is printed each wake up. This is expected.
